e3sm_to_cmip/mpas.py

```diff
@@ -437,6 +429,11 @@ def write_cmor(axes, ds, varname, varunits, d2f=True, **kwargs):
         str(varname), str(varunits), axis_ids, missing_value=fillValue, **kwargs
     )
 
+    # adding "shuffle" reduced size on disk by 14%, need to test for performance impact.
+    # TODO: Pass a commandline arg "--shuffle" down to this point.
+    # if do_shuffle and varname not in [ "lat", "lev", "lon" ]:
+    #     cmor.set_deflate(varid, True, True, 1)
+
```

Comment:
Can you make this a GH issue? That way the TODO doesn't get lost in the codebase.

Reply:
Done.